### PR TITLE
184 video slider with current time

### LINKED ISSUE #184: Add video slider with current time frame to video and edit screen

### PR DESCRIPTION
**Pull request by tfiedlerdev**
Closes #184 

# Improved label editor
image

# Improved video dialog
image

### CODE REVIEW

**Review by treyfel — DISMISSED** [review]
Looking good.

File: app/src/main/java/sensors_in_paradise/sonar/IntervalLooper.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+               if (callRunOnUiThread) {
+                   uiHandler.post(run)
+               } else {
+                    run.run()
+                }
+            }
+        }
+        timer.scheduleAtFixedRate(loopTimerTask, 0L, intervalMs)
+    }
+    fun stopLooping() {
```
Comment (lucasliebe) [inline app/src/main/java/sensors_in_paradise/sonar/IntervalLooper.kt:24]: missing empty line

File: app/src/main/java/sensors_in_paradise/sonar/IntervalLooper.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import android.os.Handler
+import android.os.Looper
+import java.util.*
+
+open class IntervalLooper(private val intervalMs: Long, val callRunOnUiThread: Boolean = false) {
+    private var loopTimerTask: TimerTask? = null
+    private val timer: Timer = Timer()
+    private val uiHandler = Handler(Looper.getMainLooper())
+    fun startLooping(run: Runnable) {
```
Comment (lucasliebe) [inline app/src/main/java/sensors_in_paradise/sonar/IntervalLooper.kt:11]: missing empty line

File: app/src/main/java/sensors_in_paradise/sonar/screen_recording/labels_editor/LabelsEditorDialog.kt [hunk 3]
```diff
@@ -262,33 +271,44 @@ class LabelsEditorDialog(
         }
         return result
     }
-
-    private fun updateRangeSlider() {
+    private fun getRangeOfCurrentAdjacentActivities(): Pair<Long, Long> {
```
Comment (lucasliebe) [inline app/src/main/java/sensors_in_paradise/sonar/screen_recording/labels_editor/LabelsEditorDialog.kt:275]: in this file are some missing empty lines

**Review by lucasliebe — APPROVED** [review]

**Pull request merged**